src/backend/marsha/core/management/commands/check_harvested.py

```diff
@@ -38,9 +39,14 @@ class Command(BaseCommand):
 
     def handle(self, *args, **options):
         """Execute management command."""
+        expired_date = generate_expired_date()
         videos = Video.objects.filter(
-            upload_state=READY,
-            updated_on__lte=generate_expired_date(),
+            Q(
+                live_state=HARVESTED,
+                upload_state=PENDING,
+            )
+            and Q(starting_at__lte=expired_date)
+            | Q(starting_at=None, uploaded_on__lte=expired_date),
```

Comment:
To be less error prone I think it is more secure to write it like this, WDYT ?

```
            Q(
                live_state=HARVESTED,
                upload_state=PENDING,
                starting_at__lte=expired_date
            )
            | Q(
               live_state=HARVESTED,
               upload_state=PENDING,
               starting_at__is_null=True, 
               uploaded_on__lte=expired_date
           ),
```

More over, the `starting_at__is_null=True` should be use instead of `starting_at=None`

Reply:
It's more readable, I agree